src/map.cpp

```diff
@@ -1795,26 +1795,29 @@ std::string map::features( const tripoint &p )
     // FIXME: can't control length of localized text.
     // Make the caller wrap properly, if it does not already.
     std::string ret;
-    if (is_bashable(p)) {
-        ret += _("Smashable. ");
+    if( is_bashable( p ) ) {
+        ret += _( "Smashable. " );
     }
-    if (has_flag("DIGGABLE", p)) {
-        ret += _("Diggable. ");
+    if( has_flag( "DIGGABLE", p ) ) {
+        ret += _( "Diggable. " );
     }
-    if (has_flag("ROUGH", p)) {
-        ret += _("Rough. ");
+    if( has_flag( "ROUGH", p ) ) {
+        ret += _( "Rough. " );
     }
-    if (has_flag("UNSTABLE", p)) {
-        ret += _("Unstable. ");
+    if( has_flag( "UNSTABLE", p ) ) {
+        ret += _( "Unstable. " );
     }
-    if (has_flag("SHARP", p)) {
-        ret += _("Sharp. ");
+    if( has_flag( "SHARP", p ) ) {
+        ret += _( "Sharp. " );
     }
-    if (has_flag("FLAT", p)) {
-        ret += _("Flat. ");
+    if( has_flag( "FLAT", p ) ) {
+        ret += _( "Flat. " );
     }
-    if (has_flag("EASY_DECONSTRUCT", p)) {
-        ret += _("Simple. ");
+    if( has_flag( "EASY_DECONSTRUCT", p ) ) {
+        ret += _( "Simple. " );
+    }
+    if( has_flag( "MOUNTABLE", p ) ) {
+        ret += _( "Mountable. " );
```

Comment:
Could you please replace the whole thing with a `stringstream` since you've touched that code anyway?

Reply:
You mean replacing 

```
std::string ret;
if( is_bashable( p ) ) {
    ret += _( "Smashable. " );
}
if( has_flag( "DIGGABLE", p ) ) {
    ret += _( "Diggable. " );
}
...
return ret;
```

to

```
std::stringstream ret;
if( is_bashable( p ) ) {
    ret << _( "Smashable. " );
}
if( has_flag( "DIGGABLE", p ) ) {
    ret << _( "Diggable. " );
}
...
return ret.str();
```

?

Reply:
Yes.